assets/scss/templates/_header.scss

```diff
@@ -234,45 +231,46 @@
     top: -7px;
   }
 
+  &.open {
+    visibility: visible;
+    opacity: 1;
+  }
+
   header {
     display: flex;
     align-items: center;
-  }
 
-  span {
-    text-overflow: ellipsis;
-    white-space: nowrap;
-    max-width: 248px;
-    display: block;
-    overflow: hidden;
-  }
+    p {
+      margin-left: 10px;
 
-  header p {
-    margin-left: 10px;
-  }
+      span {
+        @include do-ellipsis(248px, block);
```

Comment:
Are you taking advantage of the mixing here, right? ❤️

Reply:
Indeed, @andresmoschini 🙌